en_US/deploy/cluster/create-cluster.md

```diff
@@ -145,63 +147,61 @@ cluster {
 }
 ```
 
-Start all nodes one by one, and the cluster will be automatically established.
+Where, 
+
+- `discovery_strategy` is the node discovery strategy, set it to `static`
+- `seeds` is an array, where you can add the node to join the cluster, multiple nodes can be separated with `,`
 
-## Autocluster by DNS Record
+After all nodes are started, the cluster will be automatically established.
 
-### How it works
+### Autocluster by DNS Records
 
-DNS Stands for Domain Name System. A DNS server returns the IP addresses after receiving a domain query, that is, the A records. DNS system allows multiple A records for one domain name, this makes a one-to-many mapping.
+EMQX supports auto clustering by DNS A records and DNS SRV records. Domain Name System (DNS) record is a type of record used to map domain names to IP addresses. Because multiple DNS records are allowed for one domain name, EMQX leverages this feature to implement auto clustering. <!--Do we need to add more explanations here?-->
 
-EMQX can use this one-to-many mapping to find all nodes that belong to one cluster, so that the nodes can join a cluster automatically.
+#### Configure DNS Services
 
-**Configuration:** Most public cloud services have DNS services. After assigning a domain name, you only need to add the IP address of each EMQX node to the A record of this domain to finish the configuration.
+Most public cloud services have DNS services. After assigning a domain name, you only need to add the IP address of each EMQX node to the A record of this domain to finish the configuration.
 
 :::tip
 
 If EMQX is deployed in a private cloud or internal network, you will need to deploy your own DNS system, for example, with software [BIND](https://www.isc.org/bind/).
 
 :::
 
-After the preparation, we will configure in `emqx.conf` to let all nodes join the cluster using the same clustering method, `dns`.
+### Configure Autocluster by DNS Records
 
-EMQX supports DNS automatic clustering using DNS A record and DNS SRV record:
+After the DNS service is ready, you can add all nodes to join the cluster in `emqx.conf` with the `cluster.dns` configuration item:
 
-**Auto clustering using DNS A record:**
+**Example code:**
 
 ```bash
 cluster {
     discovery_strategy = dns
     dns {
         name = "localhost"
+        ## support DNS A record and DNS SRV record 
         record_type = a
     }
 }
 ```
 
-**Auto clustering using DNS SRV record:**
+Where, 
 
-```bash
-cluster {
-    discovery_strategy = dns
-    dns {
-        name = "localhost"
-        record_type = srv
-    }
-}
-```
+- `discovery_strategy` is the node discovery strategy, set it to `dns`
+- `cluster.dns.name` is a a string, input the localhost
+-  `cluster.dns.record_type` is a enum, optional value: `a` or `srv`
 
-Start all nodes one by one after the configuration, and the cluster will be automatically established.
+After all nodes are started, the cluster will be automatically established.
```

Comment:
Question: Is there a requirement for the start-up order?

Reply:
@ieQu1 could you please check the question @wivwiv left above?